packages/dds/test/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+#
+#  Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+cmake_minimum_required(VERSION 3.5.0)
+
+project(soss-dds)
+
+find_package(soss-core REQUIRED)
+find_package(xtypes REQUIRED)
+if(WIN32)
+    find_package(yaml-cpp)
+endif()
+
+########################################################################
+# Find FastRTPS
+find_package(fastrtps QUIET)
+
+if(fastrtps_DIR)
+    message(STATUS "Found a fastrtps")
+else()
+    message(STATUS "Using the fastrtps thirdparty")
+    option (THIRDPARTY "Activate internal build of dependencies" ON)
+    add_subdirectory(thirdparty/fastrtps)
```

Comment:
This is no valid now

Reply:
This `dds/test/CMakeLists.txt` is an intermediate file that should be removed... the rebase resurrected it, so I'll destroy it again :fire:.